x-pack/legacy/plugins/watcher/public/services/watches/watches.js

```diff
@@ -4,23 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import chrome from 'ui/chrome';
-import { ROUTES } from '../../../common/constants';
 import { Watch } from 'plugins/watcher/models/watch';
+import { ROUTES } from '../../../common/constants';
+import { npStart } from 'ui/new_platform';
+const { http } = npStart.core;
```

Comment:
You could make this file more migration-friendly by re-writing the `WatchesService` class using lifecycle methods, accepting `core.http` in the `setup` method, and returning the `watches` const from the setup method. But that would increase the surface area of this PR, and it looks as if this plugin hasn't been shimmed yet anyway, so it can probably wait for later.